mattermost-plugin/server/boards/configuration_test.go

```diff
@@ -77,26 +80,29 @@ func TestOnConfigurationChange(t *testing.T) {
 	t.Run("Test Load Plugin Success", func(t *testing.T) {
 		th, tearDown := SetupTestHelper(t)
 		defer tearDown()
-		api := &plugintest.API{}
-		api.On("GetUnsanitizedConfig").Return(baseConfig)
-		api.On("GetConfig").Return(baseConfig)
 
-		p := Plugin{}
-		p.SetAPI(api)
-		p.server = th.Server
-		p.wsPluginAdapter = &FakePluginAdapter{}
+		ctrl := gomock.NewController(t)
```

Comment:
We're missing a `ctrl.Finish()` here or the expectations will not get checked

Reply:
In the comments for `NewController` it says that `Finish` does not need to be called.

```
// NewController returns a new Controller. It is the preferred way to create a
// Controller.
//
// New in go1.14+, if you are passing a *testing.T into this function you no
// longer need to call ctrl.Finish() in your test methods.
```
https://github.com/golang/mock/blob/73266f9366fcf2ccef0b880618e5a9266e4136f4/gomock/controller.go#L87